### ISSUE TITLE
Mountpoint path problem in Mac OS X

### ISSUE THREAD

**Issue by pdrvsky** [resin-io-modules/drivelist#154]
Hi,

I've just started using your module, which is truly fantastic, but I encoutered one problem while plugging in one of my pendrives. It is called `EXTRNL #2` and of course it is mounted in Mac OS X with path `/Volumes/EXTRNL #2`, but after running drivelist it shows the path trimmed after space (`/Volumes/EXTRNL`). When using name without space or without hash sign it works as expected.

OS report
```
EXTRNL #2:
  Capacity:	32,01 GB (32 009 879 552 bytes)
  Available:	31,62 GB (31 617 187 840 bytes)
  Writable:	Yes
  File System:	MS-DOS FAT32
  BSD Name:	disk2s1
  Mount Point:	/Volumes/EXTRNL #2
  Content:	Windows_FAT_32
```

Module
```
{
  "device": "/dev/disk2",
  "description": "Cruzer Blade",
  "size": 32010928128,
  "mountpoints": [
    {
      "path": "/Volumes/EXTRNL"
    }
  ],
  "raw": "/dev/rdisk2",
  "protected": false,
  "system": false
}
```

**Comment by lurch** [comment]
Yup, same thing affects Linux too. Fix should be easy...

**Comment by jviotti** [comment]
I released v5.0.18 to npm. Let me know how it goes!
